src/main/java/com/example/tournamentMaker/tournament/Tournament.java

```diff
@@ -31,6 +31,7 @@ public class Tournament {
     private Type type;
     @Enumerated(EnumType.STRING)
     private Sport sport;
+    private boolean isRegistrationComplete = false;
```

Comment:
nie nazywaj zmiennej od is bo potem getter moglby wygladac isIsRegistrationComplete 
albo setIsRegistrationComplete. 
Nie wyglada to glupio ?